CMakeLists.txt

```diff
@@ -89,6 +89,7 @@ catkin_package(
 ## Your package locations should be listed before other locations
 include_directories(
   .
+  src/kits/daisy
```

Comment:
minor -- but I'm a bit uncertain why we need this here; it pollutes the include path for the other examples.  I think this could probably be fixed by changing some of the exact include paths in the source files themselves